mcs/class/System.Private.CoreLib/System.Threading/Thread.cs

```diff
@@ -4,16 +4,80 @@
 
 namespace System.Threading
 {
+	//
+	// Under netcore, there is only one thread object per thread
+	//
+	[StructLayout (LayoutKind.Sequential)]
 	partial class Thread
 	{
-		string name;
-		InternalThread internalThread;
+#pragma warning disable 169, 414, 649
```

Comment:
We cannot do that, the shared copy will get fields as well and we cannot guarantee the order.